src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/external/ResettlementPassportDeliusApiService.kt

```diff
@@ -84,7 +79,7 @@ class ResettlementPassportDeliusApiService(
 
   @Cacheable("resettlement-passport-delius-api-get-com-by-noms-id", unless = "#result == null")
   fun getComByNomsId(nomsId: String): String? {
-    val crn = findCrn(nomsId) ?: throw ResourceNotFoundException("Cannot find CRN for NomsId $nomsId in database")
+    val crn = getCrn(nomsId) ?: throw ResourceNotFoundException("Cannot find CRN for NomsId $nomsId in delius")
```

Comment:
We may need to refactor this a bit to get around this issue: https://stackoverflow.com/questions/12115996/spring-cache-cacheable-method-ignored-when-called-from-within-the-same-class

Reply:
Refactored to getCrn before calling the functions therefore renamed these to say .....ByCrn rather than .....ByNomsId